### PR TITLE
update, environments

### PR DESCRIPTION
**Pull request by genwhittTTD**
Update environments info for EUID.

### CODE REVIEW

File: docs/getting-started/gs-environments.md [hunk 1]
```diff
@@ -13,10 +13,10 @@ The following table lists all current testing and production environments for EU
 
 All EUID endpoints use the same base URL.
 
-| Environment | Base URL |
-| :--- | :--- |
-| Testing | `https://integ.euid.eu/v2` |
-| Production | `https://prod.euid.eu/v2` |
+| Environment | Cloud Region | Code | Base URL |
+| :--- | :--- | :--- | :--- |
+| Integration Testing | AWS EU West (London) | `eu-west-2.eu` | `https://integ.euid.eu/v2` |
```
Comment (asloobq) [inline docs/getting-started/gs-environments.md:18]: Hi Gen, the code is just `eu-west-2` without the `.eu`

Comment (genwhittTTD) [reply docs/getting-started/gs-environments.md]: @asloobq TYSM. Fixed.

**Review by asloobq — APPROVED** [review]

File: docs/getting-started/gs-environments.md [hunk 2]
```diff
@@ -13,10 +13,10 @@ The following table lists all current testing and production environments for EU
 
 All EUID endpoints use the same base URL.
 
-| Environment | Base URL |
-| :--- | :--- |
-| Testing | `https://integ.euid.eu/v2` |
-| Production | `https://prod.euid.eu/v2` |
+| Environment | Cloud Region | Code | Base URL |
+| :--- | :--- | :--- | :--- |
+| Integration Testing | AWS EU West (London) | `eu-west-2` | `https://integ.euid.eu/v2` |
```
Comment (sunnywu) [inline docs/getting-started/gs-environments.md:18]: minor nit: It should really be "AWS Europe (London)" according to AWS [page](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) - not "AWS EU West (London)".
For ap-northeast-1  it is AWS Asia Pacific (Tokyo) and we don't call it Asia Northwest (Tokyo) for example

Comment (genwhittTTD) [reply docs/getting-started/gs-environments.md:18]: Fixed in follow-up PR, thx.

**Pull request merged**